SparkPost/2web2ui
# SE-156: Release Recipient validation V2

### What Changed
 - Remove feature flag `recipientValidationV2`

### How To Test
 - Go to `/recipient-validation/list`
 - Check that the flow works properly without the flag

### To Do
